l10n_ec_ote/models/res_company.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    l10n_ec_parish_id = fields.Many2one(
```

Comment:
What is the purpose of this field? if you have on partner, can be retrieve with partner on company

Reply:
@cormaza is to facilitate setup for company address into company without need go to Contact menu, similar to https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/models/res_company.py#L73-L80 but as `related` with `readonly=False` instead of `compute` with `inverse`

What do you think, a better suggestion?